Directory.Packages.props

```diff
@@ -12,7 +12,7 @@
     <OpenTelemetryInstrumentationAspNetCoreLatestStableVersion>1.14.0</OpenTelemetryInstrumentationAspNetCoreLatestStableVersion>
     <OpenTelemetryInstrumentationHttpLatestStableVersion>1.14.0</OpenTelemetryInstrumentationHttpLatestStableVersion>
     <OpenTelemetryInstrumentationRuntimeLatestStableVersion>1.14.0</OpenTelemetryInstrumentationRuntimeLatestStableVersion>
-    <OpenTelemetryEnrichmentUnstableLatestVersion>1.13.0-beta.1</OpenTelemetryEnrichmentUnstableLatestVersion>
+    <OpenTelemetryEnrichmentUnstableLatestVersion>1.14.0-beta.1</OpenTelemetryEnrichmentUnstableLatestVersion>
```

Comment:
Enrichment package was not updated as a part of the release.
@evgenyfedorov2, could you please check it and prepare some fixes to the CI?